config.json

```diff
@@ -324,6 +324,10 @@
     "pdp-non-sale-price-label": "",
     "pdp-retail-price-label": "",
     "pdp-custom-fields-tab-label": "",
+    "paymentbuttons-paypal-accelerated-checkout-color": "black",
```

Comment:
Are these default colours working correct with colour picker? I thought that here should be presented valid hex values

Reply:
Yep, see screenshot
![Screenshot 2022-09-05 at 20 05 47](https://user-images.githubusercontent.com/18176525/188625058-ee7d32f7-d242-4762-9e30-e40074f8aae2.png)

Reply:
Yep, see screenshot 
![Screenshot 2022-09-06 at 14 31 26](https://user-images.githubusercontent.com/18176525/188625330-cc06b52b-7715-4c6c-8ecb-2bdce0ee5939.png)